server/utils/personUtils.ts

```diff
@@ -2,6 +2,7 @@ import dateUtils from './dateUtils'
 import stringUtils from './stringUtils'
```

Comment:
Re: the second part of the commit message: organisation utils has a present method that returns a presenter with only summary list rows added, so I think there might be a case for it here? Although in the organisation case it does incorporate the offering emails

Reply:
I'm going to leave this as-is until we add further functionality - we can expand in future if we need to.